app/src/main/java/fr/free/nrw/commons/utils/SystemThemeUtils.java

```diff
@@ -14,6 +14,10 @@
     private Context context;
     private JsonKvStore applicationKvStore;
 
+    public final String THEME_MODE_DEFAULT = "0";
```

Comment:
make these static and access them statically

Reply:
Done!